code/game/objects/structures/flora.dm

```diff
@@ -71,18 +71,18 @@ GLOBAL_LIST_EMPTY(flora_uprooting_tools_typepaths)
 	if(!required_tools)
 		required_tools = list()
 		if(flora_flags & FLORA_WOODEN)
-			required_tools += FLORA_HARVEST_WOOD_TOOLS
+			required_tools += FLORA_HARVEST_WOOD_TOOLS //This list does not include TOOL_SAW tools, they are handled seperately in can_harvest() for the purpose of on/off states
```

Comment:
Are saws still considered saws when they're off? I thought you were changing that in the files above.

Reply:
Chainsaws are but they saw slower, but the other items that use TOOL_SAW lose it when turned off. Means that they cant be used, i.e. chopping a tree with an esword hilt. I had to use this because you cant remove it from the list based on this - hence why items that use TOOL_SAW arent allowed in the list.